integration_test/uninstall_test.go

```diff
@@ -49,3 +55,22 @@ func TestKrewRemove_AliasSupported(t *testing.T) {
 	test.Krew("remove", validPlugin).RunOrFailOutput()
 	test.AssertExecutableNotInPATH("kubectl-" + validPlugin)
 }
+
+func TestKrewRemove_ManifestRemovedFromIndex(t *testing.T) {
+	skipShort(t)
+
+	test, cleanup := NewTest(t)
+	defer cleanup()
+
+	test = test.WithIndex()
+	localManifest := filepath.Join(test.Root(), "index", "plugins", validPlugin+constants.ManifestExtension)
```

Comment:
I'm a bit unhappy that we depend on the implementation detail of where the manifest is stored. It would be cleaner to use `environment.Paths` here. Maybe it's time to refactor the `environment` package, so that it can be used without setting up env variables.

Reply:
Honestly I agree, but we'd immediately know when we change the paths because the test would start breaking.

Reply:
That said I think we can expose `NewPaths()` here that doesn't rely on environment variables, for testing or other purposes:

![image](https://user-images.githubusercontent.com/159209/62244478-5d199480-b394-11e9-816d-c5e3b56cfe0d.png)

Reply:
Ok, let's do that in another PR and close this one. I think there are more locations which can be made simpler.